core/storage/story/gae_models_test.py

```diff
@@ -48,6 +48,51 @@ def test_get_deletion_policy(self) -> None:
             story_models.StoryModel.get_deletion_policy(),
             base_models.DELETION_POLICY.NOT_APPLICABLE)
 
+    def test_get_export_policy(self) -> None:
+        export_policy = story_models.StoryModel.get_export_policy()
```

Comment:
Better to create a dict (manually) and compare equality of the dict, otherwise this test is susceptible to there being extra fields in export_policy that are not being checked.

Reply:
Initially, I did create a `sample_dict = {'title': base_models.EXPORT_POLICY.NOT_APPLICABLE ....}`  like dict. However, the terminal returned failure, Thus, I shifted to this way trying to remove the issues related to extra fields.

Reply:
Done!